yaml_test.go

```diff
@@ -832,8 +832,8 @@ func TestJSONObjectToYAMLObject(t *testing.T) {
 			got := JSONObjectToYAMLObject(tt.input)
 			sortMapSlicesInPlace(tt.expected)
 			sortMapSlicesInPlace(got)
-			if !reflect.DeepEqual(got, tt.expected) {
```

Comment:
keep the comparison using reflect.DeepEqual, just use cmp.Diff for output (same comment below)

Reply:
Sure, but ... why? :-)

Reply:
because we know what reflect.DeepEqual does and it's less magical / easier to predict than a comparison library... comparison libraries sometimes mask differences or delegate equality checks to methods implemented by the object which can have unexpected results

Reply:
Fair enough!  cmp.Diff is usually what I want in a test, but I agree I didn't write this test so shouldn't mess with the logic!